azext_iot/tests/iothub/configurations/test_iot_config_int.py

```diff
@@ -165,6 +168,41 @@ def test_edge_deployments(self):
                 ],
             )
 
+            # Metrics + labels using narg kvp parameters. Configurations must be lowercase and will be lower()'ed.
+            # Note: $schema is included as a nested property in the sample content.
+            self.cmd(
+                self.set_cmd_auth_type(
+                    """iot edge deployment create -d {} --pri {} --tc \"{}\" --cl '{}' -k '{}'
+                    --cmq '{}' -n {} -g {}""".format(
+                        config_ids[5],
+                        priority,
+                        condition,
+                        "{custom_labels}",
+                        edge_content_path,
+                        "{custom_metric_queries}",
+                        self.entity_name,
+                        self.entity_rg
+                    ),
+                    auth_type=auth_phase
+                ),
+                checks=[
+                    self.check("id", config_ids[5]),
+                    self.check("priority", priority),
+                    self.check("targetCondition", condition),
+                    self.check("labels", json.loads(self.kwargs["labels"])),
```

Comment:
This should be asserting against the kvp input for labels, currently stored in `self.kwargs["custom_labels"]`